srtcore/buffer_snd.cpp

```diff
@@ -625,6 +625,16 @@ CSndBuffer::duration CSndBuffer::getBufferingDelay(const time_point& tnow) const
     return tnow - m_pFirstBlock->m_tsOriginTime;
 }
 
+CSndBuffer::time_point CSndBuffer::getFirstPacketTS() const
+{
+    ScopedLock lck(m_BufLock);
+    SRT_ASSERT(m_pFirstBlock);
+    if (m_iCount == 0)
+        return time_point(0);
```

Comment:
`time_point(0)` -> `time_point()`.
Then you'll need to check `if (first_pkt_ts != time_point())`.
But might be better to keep using the `CSndBuffer::getBufferingDelay(..)` function.